test/autoscaling.test.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const assume = require('assume');
+const async = require('async');
+const nock = require('nock');
+
+const common = require('./common');
+const beforeTesting = common.beforeTesting;
+const resourceName = 'test-name';
+
+describe('lib.autoscaling', () => {
+  describe('.autoscaling', () => {
+    const path = `/apis/autoscaling/v1/namespaces/${ common.currentName }/horizontalpodautoscalers`;
+    const resourcePath = `${ path }/${ resourceName }`;
+    const horizontalAutoscaleObj = {
+      apiVersion: 'autoscaling/v1',
+      kind: 'HorizontalPodAutoscaler',
+      metadata: {
+        name: 'test-name',
+        namespace: 'test-namespace'
```

Comment:
Hard-coding the namespace here will cause integration tests to fail.  I think you can remove this line without any ill effects and also fix integration tests.

Reply:
Namespace removed